internal/pkg/cli/deploy/deploy.go

```diff
@@ -907,6 +907,10 @@ func (d *lbSvcDeployer) stackConfiguration(in *StackRuntimeConfiguration) (*svcS
 	if err != nil {
 		return nil, err
 	}
+
+	if d.lbMft.RoutingRule.HostedZone != nil {
+		return nil, fmt.Errorf("hosted_zone is not supported for Load Balanced Web Services")
```

Comment:
```suggestion
		return nil, fmt.Errorf(`"hosted_zone" is not supported for Load Balanced Web Services`)
```

Reply:
I don't see a reason why hosted zone can't be used for LBWS. I think hosted zone can only be used when the env has imported certs, which requires alias.

Reply:
i agree there's no reason why LBWS shouldn't support `hosted_zone`. however, i just want to focus on support for backend service for now, and we can bring it to LBWS once that's working.

Reply:
discussed offline, will add lbws hosted zone support now